pkgs/tools/admin/hedgedoc-cli/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, pkgs, stdenv, fetchFromGitHub }:
+let
+  pname = "hedgedoc-cli";
+  version = "1.0";
```

Comment:
You don't need the let block. You can put those inside `mkDerivation` and use `stdenv.mkDerivation rec {` to make them available in src.